src/tests/stdlib.rs

```diff
@@ -22,6 +22,22 @@ fn mkfile() -> (PathBuf, TempDir) {
     (file_path, temp_dir)
 }
 
+fn load_stdlib_test(func: &str) -> (String, String) {
+    let mut test = String::new();
+    fs::File::open(format!("./src/tests/stdlib/{func}.ab"))
```

Comment:
there is no need to try open a non existing path. the stack above has a `_test` variable, you should unignore it and pass here

Reply:
also uhh identation should be like this to be consistent with the rest of the code

```rs
File::open()
    .expect()
    .something()
```